.gitignore

```diff
@@ -3,3 +3,4 @@
 lib/
 node_modules
 usage.txt
+.idea/
```

Comment:
I'm with @jbpros here. I believe anything that is not universal to this project belongs in your local global gitignore.